src/jit/emitarm64.cpp

```diff
@@ -11346,6 +11474,24 @@ void emitter::emitDispIns(
             emitDispReg(id->idReg3(), size, false);
             break;
 
+        case IF_DV_3F: // DV_3F   ..........mmmmm ......nnnnnddddd       Qd Sn Vm (vector)
+            emitDispReg(id->idReg1(), size, true);
+            emitDispReg(id->idReg2(), EA_4BYTE, true);
+            emitDispVectorReg(id->idReg3(), id->idInsOpt(), false);
+            break;
+
+        case IF_DV_3G: // DV_3G   ...........mmmmm ......nnnnnddddd      Vd Vn Vm (vector)
+            emitDispVectorReg(id->idReg1(), id->idInsOpt(), true);
+            emitDispVectorReg(id->idReg2(), id->idInsOpt(), true);
+            emitDispVectorReg(id->idReg3(), id->idInsOpt(), false);
+            break;
+
+        case IF_DV_3H: // DV_3H   ...........mmmmm ......nnnnnddddd      Qd Qn Vm (vector)
+            emitDispReg(id->idReg1(), size, true);
+            emitDispReg(id->idReg2(), size, true);
+            emitDispVectorReg(id->idReg3(), id->idInsOpt(), false);
+            break;
+
```

Comment:
After collapsing (DV_3F, DV_3G and DV_3H) in order to print the correct register in the Asm Dump you will need to special case using either opcode or the binary bits of 'code'.  (See IF_DR_3A for opcode or IF_DV_2O for 'code')